.github/eks-workflow-files/axlearn/axlearn-job.yml

```
@@ -15,46 +13,41 @@ spec:
                   image: PLACEHOLDER
                   command:
                     - bash
-                    - -xo
-                    - pipefail
                     - -c
                     - |
-                      test-axlearn.sh \
-                        --directory "." \
-                        --output "/opt/output/" \
-                        --test-files "/opt/axlearn/axlearn/common/*_test.py"
+                      set -exo pipefail
+                      pip install pytest-reportlog pytest-xdist
 
-                      sync  
-                      wait 
-                      # after execution flag the results have been produced 
-                      touch /opt/output/done
-                  resources:
-                    limits:
-                        nvidia.com/gpu: 8
-                  volumeMounts:
-                    - name: output
-                      mountPath: /opt/output
-                - name: upload
-                  image: amazon/aws-cli
+                      LOG_DIR="/opt/output/logs"
+                      mkdir -p ${LOG_DIR}
+
+                      # Start MPS daemon
+                      nvidia-cuda-mps-control -d &
```

```suggestion
                      nvidia-cuda-mps-control -d
```
-d will make it detach on its own